cartographer_ros/cartographer_ros/sensor_bridge.cc

```diff
@@ -181,6 +181,9 @@ void SensorBridge::HandleLaserScan(
     const std::string& sensor_id, const carto::common::Time time,
     const std::string& frame_id,
     const carto::sensor::PointCloudWithIntensities& points) {
+  if (points.points.empty()) {
+    return;
```

Comment:
super-nit: put it in one line, not 3.

Reply:
Done

Reply:
Doesn't the Google coding style mandate always putting braces, even for single-command blocks?